api_app/analyzers_manager/migrations/0119_analyzer_config_apk_artifacts.py

```diff
@@ -0,0 +1,117 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+)
+
+plugin = {
+    "python_module": {
+        "health_check_schedule": None,
+        "update_schedule": None,
+        "module": "artifacts.Artifacts",
+        "base_path": "api_app.analyzers_manager.file_analyzers",
+    },
+    "name": "APK_Artifacts",
+    "description": "[Artifacts](https://github.com/guelfoweb/artifacts) is a tool that does APK strings analysis. Useful for first analysis.",
+    "disabled": False,
+    "soft_time_limit": 60,
+    "routing_key": "default",
+    "health_check_status": True,
+    "type": "file",
+    "docker_based": False,
+    "maximum_tlp": "RED",
+    "observable_supported": [],
+    "supported_filetypes": ["application/vnd.android.package-archive"],
```

Comment:
I would set the same mimetypes that are used for APKiD - 

application/java-archive
application/vnd.android.package-archive
application/x-dex
application/zip